### PR TITLE
Fixed Grok patterns in apache tomcat integration

### PR DESCRIPTION
**Pull request by Alphayeeeet**
## Proposed commit message
Access: When parsing for example a 302 request, the response-bytes are "-" and therefore, the grok processor throwed an error. This should now be fixed.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by Alphayeeeet** [comment]
I just signed the contributer agreement.

File: packages/apache_tomcat/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.2"
+  changes:
+    - description: "Fixed some non-matching grok patterns in access log pipeline"
```
Comment (muthu-mps) [inline packages/apache_tomcat/changelog.yml:4]:
```suggestion
    - description: "Fix non-matching grok patterns in access log pipeline for 302 errors"
```

**Comment by muthu-mps** [comment]
+ Can you update the `manifest.yml` file with the latest changelog version.
+ Include the sample log format for `302 error` in the _dev/test/pipeline/test-access.log file and re-generate the expected json file.

**Comment by Alphayeeeet** [comment]
Wrong git commit username. My bad. I signed the CLA again for the other username

**Comment by Alphayeeeet** [comment]
done

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Comment by muthu-mps** [comment]
@Alphayeeeet - 

- Can you check the pipeline test execution is successful? 
`elastic-package test pipeline`
- If it fails then generate the sample document by running `elastic-package test pipeline -g` and commit the changes.

**Comment by Alphayeeeet** [comment]
@muthu-mps done. The problem was, that a non 2XX status code does not give event.outcome as success. 

I am just not sure why the pipeline does'nt report event.outcome as non-successful on 1XX or 3XX status code response. In my opinion this behaviour should be changed too, so just <=400 is event.outcome failure.

**Comment by muthu-mps** [comment]
> @muthu-mps done. The problem was, that a non 2XX status code does not give event.outcome as success.
> 
> I am just not sure why the pipeline does'nt report event.outcome as non-successful on 1XX or 3XX status code response. In my opinion this behaviour should be changed too, so just <=400 is event.outcome failure.

Thanks for catching the event.outcome parsing issue. We will update the processor to fix this.

**Pull request merged**